<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogVTKTransform</class>
 <widget class="QDialog" name="DialogVTKTransform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>664</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>375</width>
    <height>664</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>375</width>
    <height>752</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="minimumSize">
      <size>
       <width>357</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>357</width>
       <height>200</height>
      </size>
     </property>
     <property name="title">
      <string>Selected Component</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0" rowspan="2">
       <widget class="QListWidget" name="listWidget">
        <property name="minimumSize">
         <size>
          <width>299</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>299</width>
          <height>500</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="geoSelectPoint">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="toolTip">
         <string extracomment="Clicked Button Selected Components"/>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../qrc/qianfan.qrc">
          <normaloff>:/QUI/geometry/selectface.png</normaloff>:/QUI/geometry/selectface.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>8</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>357</width>
       <height>252</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>357</width>
       <height>252</height>
      </size>
     </property>
     <property name="title">
      <string>Rotate</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="rotateAngleLabel">
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>91</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Rotate Angle : </string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>65</width>
            <height>25</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonX">
          <property name="minimumSize">
           <size>
            <width>59</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>59</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>X Axis</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonY">
          <property name="minimumSize">
           <size>
            <width>59</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>59</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Y Axis</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonZ">
          <property name="minimumSize">
           <size>
            <width>59</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>59</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Z Axis</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonC">
          <property name="minimumSize">
           <size>
            <width>89</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>89</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Custom Axis</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="rotateAxisLabel">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Rotate Axis : </string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QGroupBox" name="customRotAxisGBox">
        <property name="minimumSize">
         <size>
          <width>337</width>
          <height>125</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>337</width>
          <height>125</height>
         </size>
        </property>
        <property name="title">
         <string>Custom Rotate Axis</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="customXLabel">
           <property name="minimumSize">
            <size>
             <width>96</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>96</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>Custom X Axis : </string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="customXDSB">
           <property name="minimumSize">
            <size>
             <width>212</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>212</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="customYLabel">
           <property name="minimumSize">
            <size>
             <width>96</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>96</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>Custom Y Axis : </string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="customYDSB">
           <property name="minimumSize">
            <size>
             <width>212</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>212</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="customZLabel">
           <property name="minimumSize">
            <size>
             <width>96</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>96</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>Custom Z Axis : </string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="customZDSB">
           <property name="minimumSize">
            <size>
             <width>212</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>212</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="rotateAngleDSB">
        <property name="minimumSize">
         <size>
          <width>238</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>238</width>
          <height>25</height>
         </size>
        </property>
        <property name="minimum">
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>357</width>
       <height>125</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>357</width>
       <height>125</height>
      </size>
     </property>
     <property name="title">
      <string>Move Location</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="directionXLabel">
        <property name="minimumSize">
         <size>
          <width>132</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>132</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Direction Of X Axis : </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="directionXDSB">
        <property name="minimumSize">
         <size>
          <width>196</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>196</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximum">
         <double>10000000000000000139372116959414099130712064.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="directionYLabel">
        <property name="minimumSize">
         <size>
          <width>132</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Direction Of Y Axis : </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="directionYDSB">
        <property name="minimumSize">
         <size>
          <width>196</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximum">
         <double>10000000000000000139372116959414099130712064.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="directionZLabel">
        <property name="minimumSize">
         <size>
          <width>132</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Direction Of Z Axis : </string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="directionZDSB">
        <property name="minimumSize">
         <size>
          <width>196</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="minimumSize">
      <size>
       <width>357</width>
       <height>110</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>357</width>
       <height>110</height>
      </size>
     </property>
     <property name="title">
      <string>Scale</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="scaleXLabel">
        <property name="minimumSize">
         <size>
          <width>108</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Scale Of X Axis : </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="scaleXDSB">
        <property name="minimumSize">
         <size>
          <width>221</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="scaleYLabel">
        <property name="minimumSize">
         <size>
          <width>108</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Scale Of Y Axis : </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="scaleYDSB">
        <property name="minimumSize">
         <size>
          <width>221</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="scaleZLabel">
        <property name="minimumSize">
         <size>
          <width>108</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Scale Of Z Axis : </string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="scaleZDSB">
        <property name="minimumSize">
         <size>
          <width>221</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QDialogButtonBox" name="btBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qrc/qianfan.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogVTKTransform</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogVTKTransform</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
